web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -1073,8 +1073,12 @@ fixmystreet.display = {
   reports_list: function(reportListUrl, callback) {
     // If the report list is already in the DOM,
     // just reveal it, rather than loading new page.
-    if ($('#side').length) {
-        $('#side').show();
+    var side = $('#side');
```

Comment:
Could this be `$side` to make it more obvious it's a jQuery object?

Reply:
I have used `getElementById` instead :)